src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -149,4 +164,297 @@ describe('Calendar', () => {
     expect(onSelect).toBeCalledWith(expect.stringMatching(/^2018-01-17T/));
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('first day sunday week monday', () => {
+    // the first day of the month is on Sunday but the
+    // user wants the week to start on Monday
+    const { container } = render(
+      <Grommet>
+        <Calendar
+          firstDayOfWeek={1}
+          date="2020-03-01T00:00:00-08:00"
+          animate={false}
+        />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('animate', () => {
+    jest.useFakeTimers();
+    const { container, getByLabelText } = render(
+      <Grommet>
+        <Calendar date={DATE} />
+      </Grommet>,
+    );
+    fireEvent.click(getByLabelText('December 2017'));
+    act(() => {
+      jest.advanceTimersByTime(400);
+    });
+    fireEvent.click(getByLabelText('January 2018'));
+    act(() => {
+      jest.advanceTimersByTime(400);
+    });
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
How does the snapshot taken captures the animation changes and all the onClick events?

Reply:
I changed the test so that it is testing if the month changes when the previous/next buttons are clicked and added snapshots after each that show what month is being displayed